internal/pkg/release.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

If someone comes to you from NixOS, they will have a few suggestions like these.

Reply:
Thanks!